packages/donor/src/screens/bookDonation/BookDonationScreen.tsx

```diff
@@ -67,116 +40,13 @@ export default function BookDonationScreen({
         <img src={Illustration} alt={"illustration"} />
       </div>
 
-      <div className={styles.dropdownContainer}>
-        <Select
-          analyticsName="show_appointments_in_hospital"
-          label={"הצג תורים ב:"}
-          className={styles.dropdown}
-          options={HospitalUtils.getHospitalOptions(
-            HospitalUtils.activeHospitals,
-            "הכל"
-          )}
-          value={selectedHospital}
-          onChange={setSelectedHospital}
-          isDisabled={isFetching}
-        />
-      </div>
-
-      <Popup
-        name="TooCloseAppointment"
-        title={"מועד קרוב מידי"}
-        buttonApproveText={"כן, אשר תור"}
-        open={tooCloseDonationPopupProps.open}
-        goBackText={"התחרטתי, חזרה למסך התורים"}
-        onApproved={tooCloseDonationPopupProps.onApproved}
-        onBack={tooCloseDonationPopupProps.onBack}
-        onClose={tooCloseDonationPopupProps.onClose}
-        image={WarningLogo}
-      >
-        טרם חלפו חודש ימים מיום תרומתך האחרון. האם את/ה בטוח/ה שברצונך לקבוע תור
-        זה?
-      </Popup>
-
-      <div className={styles.donationsCard}>
-        {Donations(
-          selectedHospital,
-          isFetching,
-          sortedDonationDays,
-          onSlotSelected
-        )}
-      </div>
+      <AppointmentSelect
```

Comment:
Same for this.
Can the extraction of the appointment selection to a component be in a different PR?

The reason is that reviewing a PR with 50 files and 4,601 lines changes is very hard. 

Please go over this entire pr and try to split it to smaller commits. 
Anything that is not pure Appointment Swap feature should be in a different PR.

Reply:
Rolled Back